catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -479,6 +481,15 @@ public static double round(double value, int precision) {
 		return (double) Math.round(value * scale) / scale;
 	}
 
+	private static void displayDialog(final Dialog errorDialog) {
```

Comment:
What do you think about `showDialog`? It's consistent with the dialog interface, `dialog.show()`
Btw, just name the parameter dialog, the method is not just limited to an error dialog.